index.js

```
@@ -476,8 +766,8 @@ server.setRequestHandler(CallToolRequestSchema, async (request) => {
         const graph = memoryServer.readGraph();
         return { content: [{ type: "text", text: JSON.stringify(graph) }] };
       case "search_nodes":
-        const searchResults = memoryServer.searchNodes(args.query);
-        return { content: [{ type: "text", text: JSON.stringify(searchResults) }] };
+        const searchResults = memoryServer.searchNodes(args);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)